targets/targets.json

```diff
@@ -1612,6 +1612,23 @@
         "extra_labels_add": ["DELTA_DFCM_NNN40"],
         "macros_add": ["TARGET_DELTA_DFCM_NNN40", "TARGET_NRF_LFCLK_RC"]
     },
+    "DELTA_DFCM_NNN50": {
```

Comment:
This target currently isn't being marked as a release target, is this your intention? You can add the `release_versions` field to this entry to enable this.